### PR TITLE
linked the header properly from doc page

### PR DESCRIPTION
**Pull request by lolpanda2004**
## Description

Fixed the link from the doc page and made the link connect to proper html code and redesigned it from a button to a natural looking header name like other tags

## Related Issues

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes 1050

## Type of PR

- [X] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
https://github.com/user-attachments/assets/6b9174a9-fe25-4c0f-addd-cf6883e98004

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

### CODE REVIEW

**Comment by lolpanda2004** [comment]
@varshith257 Please assign this issue and review my PR! Thank you.

File: Html-Files/Doctor Experience.html [hunk 1]
```diff
@@ -22,11 +22,11 @@
         <li><a href="../index.html">Home</a></li>
         <li><a href="../index.html#about">About</a></li>
         <li><a href="../news.html">News</a></li>
-        <a href="../dex.html"><button>Service</button></a>
+        <a href="../dex.html" style="color: #ecf0f1;">Service</a>
```
Comment (varshith257) [inline Html-Files/Doctor Experience.html:25]:
```suggestion
        <li><a href="../dex.html" style="color: #ecf0f1;">Service</a></li>
```

**Review by varshith257 — CHANGES_REQUESTED** [review]
Could you follow up with this suggestion?

File: Html-Files/Doctor Experience.html (hunk 1 above)
Comment (lolpanda2004) [reply Html-Files/Doctor Experience.html]: Done!

**Comment by lolpanda2004** [comment]
I completed please do check @varshith257

**Pull request merged**